tifeatures/layer.py

```diff
@@ -261,6 +261,19 @@ def _datetime_filter_to_sql(self, interval: List[str], dt_name: str):
                     logic.V(dt_name) < logic.S(pg_funcs.cast(end, "timestamptz")),
                 )
 
+    def _sortby(self, sortby: Optional[str]):
+        sortby = sortby.strip()
+        if sortby is None:
+            sortby = self.id_column
```

Comment:
@bitner `id_column` is set as optional in `dbmodel.Table` so mypy doesn't like that sortby can be `null`

Reply:
arrrrgy, mypy :angry:

Reply:
well mypy tells the truth, it seems possible that sortby is none and id_colum is now 🤷